astropy/modeling/parameters.py

```diff
@@ -55,6 +56,52 @@ def _tofloat(value):
     return value
 
 
+# Helpers for implementing operator overloading on Parameter
+
+def _binary_arithmetic_operation(op, right=False):
```

Comment:
Minor comment, but would call it `reflected=False`, since that is how the python documentation refers to `__radd__` etc.  https://docs.python.org/3/reference/datamodel.html#emulating-numeric-types